lib/vdsm/storage/merge.py

```diff
@@ -192,18 +192,29 @@ def _extend_base_allocation(base_vol, top_vol):
     # allocation for merging top into base.
     log.debug("Measuring sub chain top=%r base=%r",
               top_vol.volUUID, base_vol.volUUID)
-    measure = qemuimg.measure(
+    top_measure = qemuimg.measure(
         top_vol.getVolumePath(),
         format=qemuimg.FORMAT.QCOW2,
         output_format=qemuimg.FORMAT.QCOW2,
         is_block=True,
         base=base_vol.getVolumePath())
-    log.debug("Measure result: %s", measure)
+    # There may be untracked bitmaps in base if they were added after the
+    # creating the top volume. Even if this is a rare case, we need to measure
```

Comment:
There may be untracked bitmaps in base if they were left after failed
delete bitmap operation.